id,name,raw,category,processed,solution,target,version,targetnew,label
5520,3560,"2022-08-26 03:48:05,026 | Failed at Play [testbed_deploy_nimbus] *********************
2022-08-26 03:48:05,026 | TASK [testbed_deploy_nimbus][Testbed deploy result is not PASS] 
task path: /home/worker/workspace/Ansible_Cycle_Windows_11_64/newgos_testing_internal/testbed_deploy/parse_testbed_info.yml:10
fatal: [localhost]: FAILED! => Get deployment result in testbedInfo.json is 'INVALID', not PASS",,timestamp failed at play testbed deploy nimbus timestamp task testbed deploy nimbus testbed deploy result is not pass task path home worker workspace ansible cycle windows number new gos testing internal testbed deploy parse testbed info yml number fatal localhost failed get deployment result in testbed info json is invalid not pass,retry,nimbus,202209081634,environment,relevant
5521,3864,"2022-09-05 04:36:07,005 | Failed at Play [deploy_vm_efi_sata_e1000e] *****************
2022-09-05 04:36:07,005 | TASK [deploy_vm_efi_sata_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70U1_SATA_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:14,005 | TASK [deploy_vm_efi_sata_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70U1_SATA_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_sata_e1000e",,timestamp failed at play deploy vm efi sata timestamp task deploy vm efi sata add virtual tpm device to vm task path home worker workspace ansible windows number sata efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi sata testing exit due to failure task path home worker workspace ansible windows number sata efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi sata e number e,deepdive,testbed,202209081634,component,relevant
5522,3862,"2022-09-05 04:36:25,005 | Failed at Play [deploy_vm_efi_lsilogicsas_e1000e] **********
2022-09-05 04:36:25,005 | TASK [deploy_vm_efi_lsilogicsas_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70GA_LSILOGICSAS_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:32,005 | TASK [deploy_vm_efi_lsilogicsas_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70GA_LSILOGICSAS_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_lsilogicsas_e1000e",,timestamp failed at play deploy vm efi lsi logic sas timestamp task deploy vm efi lsi logic sas add virtual tpm device to vm task path home worker workspace ansible windows number ga lsi logic sas efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi lsi logic sas testing exit due to failure task path home worker workspace ansible windows number ga lsi logic sas efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi lsi logic sas e number e,deepdive,testbed,202209081634,component,relevant
5523,3863,"2022-09-05 04:36:30,005 | Failed at Play [deploy_vm_efi_paravirtual_e1000e] **********
2022-09-05 04:36:30,005 | TASK [deploy_vm_efi_paravirtual_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70U3_PARAVIRTUAL_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:37,005 | TASK [deploy_vm_efi_paravirtual_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70U3_PARAVIRTUAL_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_e1000e",,timestamp failed at play deploy vm efi para virtual timestamp task deploy vm efi para virtual add virtual tpm device to vm task path home worker workspace ansible windows number para virtual efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi para virtual testing exit due to failure task path home worker workspace ansible windows number para virtual efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual e number e,deepdive,testbed,202209081634,component,relevant
5524,3670,"2022-08-31 02:17:20,031 | Failed at Play [ovt_verify_status] *************************
2022-08-31 02:17:20,031 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,machine,relevant
5525,3663,"2022-08-30 12:44:30,030 | Failed at Play [ovt_verify_status] *************************
2022-08-30 12:44:30,030 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,machine,relevant
5526,3657,"2022-08-30 10:32:36,030 | Failed at Play [ovt_verify_status] *************************
2022-08-30 10:32:36,030 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,machine,relevant
5520,3560,"2022-08-26 03:48:05,026 | Failed at Play [testbed_deploy_nimbus] *********************
2022-08-26 03:48:05,026 | TASK [testbed_deploy_nimbus][Testbed deploy result is not PASS] 
task path: /home/worker/workspace/Ansible_Cycle_Windows_11_64/newgos_testing_internal/testbed_deploy/parse_testbed_info.yml:10
fatal: [localhost]: FAILED! => Get deployment result in testbedInfo.json is 'INVALID', not PASS",,timestamp failed at play testbed deploy nimbus timestamp task testbed deploy nimbus testbed deploy result is not pass task path home worker workspace ansible cycle windows number new gos testing internal testbed deploy parse testbed info yml number fatal localhost failed get deployment result in testbed info json is invalid not pass,retry,nimbus,202209081634,component,irrelevant
5520,3560,"2022-08-26 03:48:05,026 | Failed at Play [testbed_deploy_nimbus] *********************
2022-08-26 03:48:05,026 | TASK [testbed_deploy_nimbus][Testbed deploy result is not PASS] 
task path: /home/worker/workspace/Ansible_Cycle_Windows_11_64/newgos_testing_internal/testbed_deploy/parse_testbed_info.yml:10
fatal: [localhost]: FAILED! => Get deployment result in testbedInfo.json is 'INVALID', not PASS",,timestamp failed at play testbed deploy nimbus timestamp task testbed deploy nimbus testbed deploy result is not pass task path home worker workspace ansible cycle windows number new gos testing internal testbed deploy parse testbed info yml number fatal localhost failed get deployment result in testbed info json is invalid not pass,retry,nimbus,202209081634,infrastructure,irrelevant
5520,3560,"2022-08-26 03:48:05,026 | Failed at Play [testbed_deploy_nimbus] *********************
2022-08-26 03:48:05,026 | TASK [testbed_deploy_nimbus][Testbed deploy result is not PASS] 
task path: /home/worker/workspace/Ansible_Cycle_Windows_11_64/newgos_testing_internal/testbed_deploy/parse_testbed_info.yml:10
fatal: [localhost]: FAILED! => Get deployment result in testbedInfo.json is 'INVALID', not PASS",,timestamp failed at play testbed deploy nimbus timestamp task testbed deploy nimbus testbed deploy result is not pass task path home worker workspace ansible cycle windows number new gos testing internal testbed deploy parse testbed info yml number fatal localhost failed get deployment result in testbed info json is invalid not pass,retry,nimbus,202209081634,user,irrelevant
5520,3560,"2022-08-26 03:48:05,026 | Failed at Play [testbed_deploy_nimbus] *********************
2022-08-26 03:48:05,026 | TASK [testbed_deploy_nimbus][Testbed deploy result is not PASS] 
task path: /home/worker/workspace/Ansible_Cycle_Windows_11_64/newgos_testing_internal/testbed_deploy/parse_testbed_info.yml:10
fatal: [localhost]: FAILED! => Get deployment result in testbedInfo.json is 'INVALID', not PASS",,timestamp failed at play testbed deploy nimbus timestamp task testbed deploy nimbus testbed deploy result is not pass task path home worker workspace ansible cycle windows number new gos testing internal testbed deploy parse testbed info yml number fatal localhost failed get deployment result in testbed info json is invalid not pass,retry,nimbus,202209081634,test,irrelevant
5520,3560,"2022-08-26 03:48:05,026 | Failed at Play [testbed_deploy_nimbus] *********************
2022-08-26 03:48:05,026 | TASK [testbed_deploy_nimbus][Testbed deploy result is not PASS] 
task path: /home/worker/workspace/Ansible_Cycle_Windows_11_64/newgos_testing_internal/testbed_deploy/parse_testbed_info.yml:10
fatal: [localhost]: FAILED! => Get deployment result in testbedInfo.json is 'INVALID', not PASS",,timestamp failed at play testbed deploy nimbus timestamp task testbed deploy nimbus testbed deploy result is not pass task path home worker workspace ansible cycle windows number new gos testing internal testbed deploy parse testbed info yml number fatal localhost failed get deployment result in testbed info json is invalid not pass,retry,nimbus,202209081634,machine,irrelevant
5520,3560,"2022-08-26 03:48:05,026 | Failed at Play [testbed_deploy_nimbus] *********************
2022-08-26 03:48:05,026 | TASK [testbed_deploy_nimbus][Testbed deploy result is not PASS] 
task path: /home/worker/workspace/Ansible_Cycle_Windows_11_64/newgos_testing_internal/testbed_deploy/parse_testbed_info.yml:10
fatal: [localhost]: FAILED! => Get deployment result in testbedInfo.json is 'INVALID', not PASS",,timestamp failed at play testbed deploy nimbus timestamp task testbed deploy nimbus testbed deploy result is not pass task path home worker workspace ansible cycle windows number new gos testing internal testbed deploy parse testbed info yml number fatal localhost failed get deployment result in testbed info json is invalid not pass,retry,nimbus,202209081634,product,irrelevant
5521,3864,"2022-09-05 04:36:07,005 | Failed at Play [deploy_vm_efi_sata_e1000e] *****************
2022-09-05 04:36:07,005 | TASK [deploy_vm_efi_sata_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70U1_SATA_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:14,005 | TASK [deploy_vm_efi_sata_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70U1_SATA_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_sata_e1000e",,timestamp failed at play deploy vm efi sata timestamp task deploy vm efi sata add virtual tpm device to vm task path home worker workspace ansible windows number sata efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi sata testing exit due to failure task path home worker workspace ansible windows number sata efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi sata e number e,deepdive,testbed,202209081634,environment,irrelevant
5521,3864,"2022-09-05 04:36:07,005 | Failed at Play [deploy_vm_efi_sata_e1000e] *****************
2022-09-05 04:36:07,005 | TASK [deploy_vm_efi_sata_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70U1_SATA_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:14,005 | TASK [deploy_vm_efi_sata_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70U1_SATA_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_sata_e1000e",,timestamp failed at play deploy vm efi sata timestamp task deploy vm efi sata add virtual tpm device to vm task path home worker workspace ansible windows number sata efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi sata testing exit due to failure task path home worker workspace ansible windows number sata efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi sata e number e,deepdive,testbed,202209081634,infrastructure,irrelevant
5521,3864,"2022-09-05 04:36:07,005 | Failed at Play [deploy_vm_efi_sata_e1000e] *****************
2022-09-05 04:36:07,005 | TASK [deploy_vm_efi_sata_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70U1_SATA_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:14,005 | TASK [deploy_vm_efi_sata_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70U1_SATA_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_sata_e1000e",,timestamp failed at play deploy vm efi sata timestamp task deploy vm efi sata add virtual tpm device to vm task path home worker workspace ansible windows number sata efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi sata testing exit due to failure task path home worker workspace ansible windows number sata efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi sata e number e,deepdive,testbed,202209081634,user,irrelevant
5521,3864,"2022-09-05 04:36:07,005 | Failed at Play [deploy_vm_efi_sata_e1000e] *****************
2022-09-05 04:36:07,005 | TASK [deploy_vm_efi_sata_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70U1_SATA_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:14,005 | TASK [deploy_vm_efi_sata_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70U1_SATA_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_sata_e1000e",,timestamp failed at play deploy vm efi sata timestamp task deploy vm efi sata add virtual tpm device to vm task path home worker workspace ansible windows number sata efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi sata testing exit due to failure task path home worker workspace ansible windows number sata efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi sata e number e,deepdive,testbed,202209081634,test,irrelevant
5521,3864,"2022-09-05 04:36:07,005 | Failed at Play [deploy_vm_efi_sata_e1000e] *****************
2022-09-05 04:36:07,005 | TASK [deploy_vm_efi_sata_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70U1_SATA_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:14,005 | TASK [deploy_vm_efi_sata_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70U1_SATA_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_sata_e1000e",,timestamp failed at play deploy vm efi sata timestamp task deploy vm efi sata add virtual tpm device to vm task path home worker workspace ansible windows number sata efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi sata testing exit due to failure task path home worker workspace ansible windows number sata efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi sata e number e,deepdive,testbed,202209081634,machine,irrelevant
5521,3864,"2022-09-05 04:36:07,005 | Failed at Play [deploy_vm_efi_sata_e1000e] *****************
2022-09-05 04:36:07,005 | TASK [deploy_vm_efi_sata_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70U1_SATA_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:14,005 | TASK [deploy_vm_efi_sata_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70U1_SATA_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_sata_e1000e",,timestamp failed at play deploy vm efi sata timestamp task deploy vm efi sata add virtual tpm device to vm task path home worker workspace ansible windows number sata efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi sata testing exit due to failure task path home worker workspace ansible windows number sata efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi sata e number e,deepdive,testbed,202209081634,product,irrelevant
5522,3862,"2022-09-05 04:36:25,005 | Failed at Play [deploy_vm_efi_lsilogicsas_e1000e] **********
2022-09-05 04:36:25,005 | TASK [deploy_vm_efi_lsilogicsas_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70GA_LSILOGICSAS_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:32,005 | TASK [deploy_vm_efi_lsilogicsas_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70GA_LSILOGICSAS_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_lsilogicsas_e1000e",,timestamp failed at play deploy vm efi lsi logic sas timestamp task deploy vm efi lsi logic sas add virtual tpm device to vm task path home worker workspace ansible windows number ga lsi logic sas efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi lsi logic sas testing exit due to failure task path home worker workspace ansible windows number ga lsi logic sas efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi lsi logic sas e number e,deepdive,testbed,202209081634,environment,irrelevant
5522,3862,"2022-09-05 04:36:25,005 | Failed at Play [deploy_vm_efi_lsilogicsas_e1000e] **********
2022-09-05 04:36:25,005 | TASK [deploy_vm_efi_lsilogicsas_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70GA_LSILOGICSAS_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:32,005 | TASK [deploy_vm_efi_lsilogicsas_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70GA_LSILOGICSAS_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_lsilogicsas_e1000e",,timestamp failed at play deploy vm efi lsi logic sas timestamp task deploy vm efi lsi logic sas add virtual tpm device to vm task path home worker workspace ansible windows number ga lsi logic sas efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi lsi logic sas testing exit due to failure task path home worker workspace ansible windows number ga lsi logic sas efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi lsi logic sas e number e,deepdive,testbed,202209081634,infrastructure,irrelevant
5522,3862,"2022-09-05 04:36:25,005 | Failed at Play [deploy_vm_efi_lsilogicsas_e1000e] **********
2022-09-05 04:36:25,005 | TASK [deploy_vm_efi_lsilogicsas_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70GA_LSILOGICSAS_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:32,005 | TASK [deploy_vm_efi_lsilogicsas_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70GA_LSILOGICSAS_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_lsilogicsas_e1000e",,timestamp failed at play deploy vm efi lsi logic sas timestamp task deploy vm efi lsi logic sas add virtual tpm device to vm task path home worker workspace ansible windows number ga lsi logic sas efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi lsi logic sas testing exit due to failure task path home worker workspace ansible windows number ga lsi logic sas efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi lsi logic sas e number e,deepdive,testbed,202209081634,user,irrelevant
5522,3862,"2022-09-05 04:36:25,005 | Failed at Play [deploy_vm_efi_lsilogicsas_e1000e] **********
2022-09-05 04:36:25,005 | TASK [deploy_vm_efi_lsilogicsas_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70GA_LSILOGICSAS_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:32,005 | TASK [deploy_vm_efi_lsilogicsas_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70GA_LSILOGICSAS_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_lsilogicsas_e1000e",,timestamp failed at play deploy vm efi lsi logic sas timestamp task deploy vm efi lsi logic sas add virtual tpm device to vm task path home worker workspace ansible windows number ga lsi logic sas efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi lsi logic sas testing exit due to failure task path home worker workspace ansible windows number ga lsi logic sas efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi lsi logic sas e number e,deepdive,testbed,202209081634,test,irrelevant
5522,3862,"2022-09-05 04:36:25,005 | Failed at Play [deploy_vm_efi_lsilogicsas_e1000e] **********
2022-09-05 04:36:25,005 | TASK [deploy_vm_efi_lsilogicsas_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70GA_LSILOGICSAS_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:32,005 | TASK [deploy_vm_efi_lsilogicsas_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70GA_LSILOGICSAS_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_lsilogicsas_e1000e",,timestamp failed at play deploy vm efi lsi logic sas timestamp task deploy vm efi lsi logic sas add virtual tpm device to vm task path home worker workspace ansible windows number ga lsi logic sas efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi lsi logic sas testing exit due to failure task path home worker workspace ansible windows number ga lsi logic sas efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi lsi logic sas e number e,deepdive,testbed,202209081634,machine,irrelevant
5522,3862,"2022-09-05 04:36:25,005 | Failed at Play [deploy_vm_efi_lsilogicsas_e1000e] **********
2022-09-05 04:36:25,005 | TASK [deploy_vm_efi_lsilogicsas_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70GA_LSILOGICSAS_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:32,005 | TASK [deploy_vm_efi_lsilogicsas_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70GA_LSILOGICSAS_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_lsilogicsas_e1000e",,timestamp failed at play deploy vm efi lsi logic sas timestamp task deploy vm efi lsi logic sas add virtual tpm device to vm task path home worker workspace ansible windows number ga lsi logic sas efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi lsi logic sas testing exit due to failure task path home worker workspace ansible windows number ga lsi logic sas efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi lsi logic sas e number e,deepdive,testbed,202209081634,product,irrelevant
5523,3863,"2022-09-05 04:36:30,005 | Failed at Play [deploy_vm_efi_paravirtual_e1000e] **********
2022-09-05 04:36:30,005 | TASK [deploy_vm_efi_paravirtual_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70U3_PARAVIRTUAL_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:37,005 | TASK [deploy_vm_efi_paravirtual_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70U3_PARAVIRTUAL_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_e1000e",,timestamp failed at play deploy vm efi para virtual timestamp task deploy vm efi para virtual add virtual tpm device to vm task path home worker workspace ansible windows number para virtual efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi para virtual testing exit due to failure task path home worker workspace ansible windows number para virtual efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual e number e,deepdive,testbed,202209081634,environment,irrelevant
5523,3863,"2022-09-05 04:36:30,005 | Failed at Play [deploy_vm_efi_paravirtual_e1000e] **********
2022-09-05 04:36:30,005 | TASK [deploy_vm_efi_paravirtual_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70U3_PARAVIRTUAL_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:37,005 | TASK [deploy_vm_efi_paravirtual_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70U3_PARAVIRTUAL_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_e1000e",,timestamp failed at play deploy vm efi para virtual timestamp task deploy vm efi para virtual add virtual tpm device to vm task path home worker workspace ansible windows number para virtual efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi para virtual testing exit due to failure task path home worker workspace ansible windows number para virtual efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual e number e,deepdive,testbed,202209081634,infrastructure,irrelevant
5523,3863,"2022-09-05 04:36:30,005 | Failed at Play [deploy_vm_efi_paravirtual_e1000e] **********
2022-09-05 04:36:30,005 | TASK [deploy_vm_efi_paravirtual_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70U3_PARAVIRTUAL_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:37,005 | TASK [deploy_vm_efi_paravirtual_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70U3_PARAVIRTUAL_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_e1000e",,timestamp failed at play deploy vm efi para virtual timestamp task deploy vm efi para virtual add virtual tpm device to vm task path home worker workspace ansible windows number para virtual efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi para virtual testing exit due to failure task path home worker workspace ansible windows number para virtual efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual e number e,deepdive,testbed,202209081634,user,irrelevant
5523,3863,"2022-09-05 04:36:30,005 | Failed at Play [deploy_vm_efi_paravirtual_e1000e] **********
2022-09-05 04:36:30,005 | TASK [deploy_vm_efi_paravirtual_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70U3_PARAVIRTUAL_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:37,005 | TASK [deploy_vm_efi_paravirtual_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70U3_PARAVIRTUAL_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_e1000e",,timestamp failed at play deploy vm efi para virtual timestamp task deploy vm efi para virtual add virtual tpm device to vm task path home worker workspace ansible windows number para virtual efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi para virtual testing exit due to failure task path home worker workspace ansible windows number para virtual efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual e number e,deepdive,testbed,202209081634,test,irrelevant
5523,3863,"2022-09-05 04:36:30,005 | Failed at Play [deploy_vm_efi_paravirtual_e1000e] **********
2022-09-05 04:36:30,005 | TASK [deploy_vm_efi_paravirtual_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70U3_PARAVIRTUAL_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:37,005 | TASK [deploy_vm_efi_paravirtual_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70U3_PARAVIRTUAL_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_e1000e",,timestamp failed at play deploy vm efi para virtual timestamp task deploy vm efi para virtual add virtual tpm device to vm task path home worker workspace ansible windows number para virtual efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi para virtual testing exit due to failure task path home worker workspace ansible windows number para virtual efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual e number e,deepdive,testbed,202209081634,machine,irrelevant
5523,3863,"2022-09-05 04:36:30,005 | Failed at Play [deploy_vm_efi_paravirtual_e1000e] **********
2022-09-05 04:36:30,005 | TASK [deploy_vm_efi_paravirtual_e1000e][Add virtual TPM device to VM] 
task path: /home/worker/workspace/Ansible_Windows_11_70U3_PARAVIRTUAL_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_add_remove_vtpm.yml:14
exception in /vmware_guest_tpm.py when vtpm_operation in /vmware.py when wait_for_task
fatal: [localhost]: FAILED! => Failed to configure vTPM device on virtual machine due to '('RuntimeFault.summary', None)'
2022-09-05 04:36:37,005 | TASK [deploy_vm_efi_paravirtual_e1000e][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Windows_11_70U3_PARAVIRTUAL_E1000E_EFI/ansible-vsphere-gos-validation/common/test_rescue.yml:57
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_efi_paravirtual_e1000e",,timestamp failed at play deploy vm efi para virtual timestamp task deploy vm efi para virtual add virtual tpm device to vm task path home worker workspace ansible windows number para virtual efi ansible vsphere gos validation common vm add remove vtpm yml number exception in vmware guest tpm python when vtpm operation in vmware python when wait for task fatal localhost failed failed to configure vtpm device on virtual machine due to runtime fault summary none timestamp task deploy vm efi para virtual testing exit due to failure task path home worker workspace ansible windows number para virtual efi ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm efi para virtual e number e,deepdive,testbed,202209081634,product,irrelevant
5524,3670,"2022-08-31 02:17:20,031 | Failed at Play [ovt_verify_status] *************************
2022-08-31 02:17:20,031 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,environment,irrelevant
5524,3670,"2022-08-31 02:17:20,031 | Failed at Play [ovt_verify_status] *************************
2022-08-31 02:17:20,031 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,component,irrelevant
5524,3670,"2022-08-31 02:17:20,031 | Failed at Play [ovt_verify_status] *************************
2022-08-31 02:17:20,031 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,infrastructure,irrelevant
5524,3670,"2022-08-31 02:17:20,031 | Failed at Play [ovt_verify_status] *************************
2022-08-31 02:17:20,031 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,user,irrelevant
5524,3670,"2022-08-31 02:17:20,031 | Failed at Play [ovt_verify_status] *************************
2022-08-31 02:17:20,031 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,test,irrelevant
5524,3670,"2022-08-31 02:17:20,031 | Failed at Play [ovt_verify_status] *************************
2022-08-31 02:17:20,031 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,product,irrelevant
5525,3663,"2022-08-30 12:44:30,030 | Failed at Play [ovt_verify_status] *************************
2022-08-30 12:44:30,030 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,environment,irrelevant
5525,3663,"2022-08-30 12:44:30,030 | Failed at Play [ovt_verify_status] *************************
2022-08-30 12:44:30,030 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,component,irrelevant
5525,3663,"2022-08-30 12:44:30,030 | Failed at Play [ovt_verify_status] *************************
2022-08-30 12:44:30,030 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,infrastructure,irrelevant
5525,3663,"2022-08-30 12:44:30,030 | Failed at Play [ovt_verify_status] *************************
2022-08-30 12:44:30,030 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,user,irrelevant
5525,3663,"2022-08-30 12:44:30,030 | Failed at Play [ovt_verify_status] *************************
2022-08-30 12:44:30,030 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,test,irrelevant
5525,3663,"2022-08-30 12:44:30,030 | Failed at Play [ovt_verify_status] *************************
2022-08-30 12:44:30,030 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,product,irrelevant
5526,3657,"2022-08-30 10:32:36,030 | Failed at Play [ovt_verify_status] *************************
2022-08-30 10:32:36,030 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,environment,irrelevant
5526,3657,"2022-08-30 10:32:36,030 | Failed at Play [ovt_verify_status] *************************
2022-08-30 10:32:36,030 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,component,irrelevant
5526,3657,"2022-08-30 10:32:36,030 | Failed at Play [ovt_verify_status] *************************
2022-08-30 10:32:36,030 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,infrastructure,irrelevant
5526,3657,"2022-08-30 10:32:36,030 | Failed at Play [ovt_verify_status] *************************
2022-08-30 10:32:36,030 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,user,irrelevant
5526,3657,"2022-08-30 10:32:36,030 | Failed at Play [ovt_verify_status] *************************
2022-08-30 10:32:36,030 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,test,irrelevant
5526,3657,"2022-08-30 10:32:36,030 | Failed at Play [ovt_verify_status] *************************
2022-08-30 10:32:36,030 | TASK [Check process 'vmtoolsd -n vmusr' is running] ********
task path: /home/worker/workspace/Ansible_Debian_10.x_64bit_70GA_IDE_VMXNET3_EFI/ansible-vsphere-gos-validation/linux/utils/check_process_status.yml:34
fatal: [localhost]: FAILED! => User 'vmware' doesn't have running process 'vmtoolsd -n vmusr'",,timestamp failed at play ovt verify status timestamp task check process vmtoolsd nvm user is running task path home worker workspace ansible debian number x number b it number ga ide vmxnet number efi ansible vsphere gos validation linux utilities check process status yml number fatal localhost failed user vmware doesn t have running process vmtoolsd nvm user,deepdive,targetvm,202209081634,product,irrelevant
